reward/reward_distributor_test.go

```diff
@@ -17,15 +17,104 @@
 package reward
 
 import (
+	"fmt"
 	"math/big"
 	"testing"
 
 	"github.com/klaytn/klaytn/blockchain/types"
 	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/consensus/istanbul"
 	"github.com/klaytn/klaytn/params"
 	"github.com/stretchr/testify/assert"
 )
 
+type testGovernance struct {
```

Comment:
Please move it to staking_manager_test.go. testGovernance is not used in reward_distributor_test.go.

Reply:
fixed in 3693c7c1